emr-user-role-mapper-s3storagebasedauthorizationmanager/README.md

```diff
@@ -0,0 +1,36 @@
+# S3 storage based authorization manager for Hive Metastore service.
```

Comment:
Recently when I was working with a Hive project called standalone Hive metastore: https://github.com/apache/hive/tree/master/standalone-metastore. I found out this plugin is not gonna work with the current config since standalone HMS does not even have hive.ql related packages in it. Not sure if we want to call this out

Reply:
I apologize for the back and forth. I assumed when you said Standalone service, that you meant remote vs embedded mode for HMS. The standalone service does look like its missing all the authorization pieces but I need confirm from the limited deep dive that I have done. For now, we can put it in our README file as a limitation.